README.md

```diff
@@ -164,6 +163,31 @@ plugins: [
 ]
 ```
 
+## Events
+
+To allow other plugins catch and customise the result of Favicons Library before this plugin emits all the assets, this plugin executes the following events:
+
+### AsyncSeriesWaterfallHook
+
+* `webappWebpackPluginBeforeEmit`
+
+Example implementation:
+
+````js
+  new class {
+    apply(compiler) {
+      compiler.hooks.make.tapAsync("A", (compilation, callback) => {
+        compilation.hooks.webappWebpackPluginBeforeEmit.tapAsync("B", (result, callback) => {
+          console.log(result);
+          return callback(null, result);
```

Comment:
could you add some comments making it explicit that `result` might be modified here?